tests/Fixtures/Asset/StubAssetManager.php

```
@@ -6,63 +6,55 @@
 use RuntimeException;
 
-/**
- * Stub of AssetManagerInterface for tests.
- *
- * @copyright Copyright (C) 2026 Terabytesoftw.
- * @license https://opensource.org/license/mit/ MIT License
- */
 final class StubAssetManager implements AssetManagerInterface
 {
     public function __construct(
         IOInterface $io,
```

PHPMD (2.15.0)

Avoid variables with short names like $io. Configured minimum length is 3. (undefined)

(ShortVariable)